src/context/CardContext.js

```diff
@@ -0,0 +1,16 @@
+import { createContext, useState } from 'react';
+
+const UserContext = createContext();
```

Comment:
why did you call it UserContext? The file is CardContext and that describes the nature of the context much more clearly that UserContext — there are no users in this app